RevenueCat/Scripts/StoreTransaction.cs

```diff
@@ -52,19 +52,23 @@ public class StoreTransaction
         public StoreTransaction(JSONNode response)
         {
             TransactionIdentifier = response["transactionIdentifier"];
-            RevenueCatId = response["transactionIdentifier"];
             ProductIdentifier = response["productIdentifier"];
+            #pragma warning disable 618 // Disable Obsolete warning
```

Comment:
This isn't related to `usesStoreKit2IfAvailable`, but related to #145 . This just disables some warnings that were showing in the Unity UI after that change since `RevenueCatId` and `ProductId` were marked as `Obsolete`.